cluster/etcd/stack.yaml

```diff
@@ -78,7 +75,7 @@ Resources:
         Timeout: PT15M
     Properties:
       DesiredCapacity: {{.Cluster.ConfigItems.etcd_instance_count}}
-      HealthCheckGracePeriod: 300
+      HealthCheckGracePeriod: 0
```

Comment:
Any specific reason to set this to 0? Are we relying on the `HeartbeatTimeout` in `AutoscalingLifecycleHook` instead? 🤔

Reply:
Yes, exactly that :slightly_smiling_face: